src/event.cpp

```diff
@@ -99,6 +99,7 @@ std::string enum_to_string<event_type>( event_type data )
         case event_type::throws_up: return "throws_up";
         case event_type::triggers_alarm: return "triggers_alarm";
         case event_type::uses_debug_menu: return "uses_debug_menu";
+        case event_type::u_var_changed: return "u_var_changed";
```

Comment:
Is the indentation right here?

Reply:
Astyle says it's fine.

Reply:
Looked wrong on my phone but looks fine on the regular web interface, so never mind.